iroh/tests/provide.rs

```diff
@@ -894,7 +894,7 @@ async fn test_token_passthrough() -> Result<()> {
                     if let iroh_bytes::provider::Event::GetRequestReceived { token: tok, .. } =
                         bp_msg
                     {
-                        events_sender.send(tok).ok();
+                        events_sender.send(tok).expect("receiver dropped");
```

Comment:
Why does this fix the test?

Reply:
it doesn't.  as per description it ensures we get a panic if this ever is an issue in one of the flakes.  this is more belt-and-braces as i expect the timeout to work, but with flaky tests you can't be explicit enough.